src/client/src/ui/analytics/AnalyticsContainer.tsx

```diff
@@ -1,13 +1,9 @@
-import * as PropTypes from 'prop-types'
 import * as React from 'react'
```

Comment:
Consider turning on `esModuleInterop` and `allowSyntheticDefaultImports` in tsconfig.json, then you can import React and (and other commonjs modules) like so:
```
import React from 'react'
```

Reply:
Wow. Thanks!